.github/workflows/deploy.yml

```diff
@@ -11,11 +11,17 @@ on:
           - staging
           - prod
         default: 'intg'
-      to-deploy:
-        description: 'Version to deploy'
+      keycloak-to-deploy:
+        description: 'Keycloak user version to deploy'
+        required: true
+      db-to-deploy:
+        description: 'DB users version to deploy'
+        required: true 
+      bastion-to-deploy:
+        description: 'Bastion user version to deploy'
         required: true
 
-run-name: Deploying Create Database Users Lambda ${{inputs.to-deploy}} to ${{inputs.environment}}
+run-name: Deploying Create Database Users Lambdas ${{inputs.keycloak-to-deploy}}, ${{inputs.keycloak-to-deploy}} and ${{inputs.bastion-to-deploy}} to ${{inputs.environment}}
```

Comment:
Should this be `run-name: Deploying ... Lambdas ${{inputs.keycloak-to-deploy}}, ${{inputs.db-to-deploy}} and ...`